spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ModuleRegistrar.java

```diff
@@ -31,19 +37,37 @@
  * @author Marius Bogoevici
  * @author Dave Syer
  */
-public class ModuleRegistrar implements ImportBeanDefinitionRegistrar {
+@Configuration
```

Comment:
This class doesn't necessarily need to be be a `Configuration` class. If the intention to make this a `Configuration` class is to invoke aware methods, then I don't think that is needed as the `ImportDefinitionRegistrar`'s *aware methods are invoked [when the registrar is imported] (https://github.com/spring-projects/spring-framework/blob/master/spring-context/src/main/java/org/springframework/context/annotation/ConfigurationClassParser.java#L501).